app/src/main/java/de/westnordost/streetcomplete/quests/orchard_produce/AddOrchardProduce.kt

```diff
@@ -15,6 +16,8 @@ class AddOrchardProduce : OsmFilterQuestType<List<OrchardProduce>>() {
     override val wikiLink = "Tag:landuse=orchard"
     override val icon = R.drawable.ic_quest_apple
 
+    override val questTypeAchievements = emptyList<QuestTypeAchievements>()
```

Comment:
Maybe add to `outdoors` achievement?

Reply:
yeah. There is no farmers achievement, so this will do